README.md

```diff
@@ -183,6 +183,26 @@ if (cc != NULL && cc->err) {
 }
 ```
 
+There is a hook to get notified about connect and reconnect attempts.
+This is useful for applying socket options or access endpoint information for a connection to a particular node.
+The callback is registered using the following function:
+
+```c
+int redisClusterSetConnectCallback(redisClusterContext *cc,
+                                   void(fn)(const redisContext *c, int status));
+```
+
+The callback is called just after connect, before TLS handshake and Redis authentication.
```

Comment:
Good that its before TLS! Was thinking of this case this morning as a possible problem, but you got it covered already. Nice

Reply:
What would be the problem of doing it after the TLS handshake?

Reply:
Not sure, but it seems better to be able to set timer options before TLS sends its handshake packets.
Maybe this procedure can't get stuck anyway..but now we might be able to set timeouts.